cmd/mailway/main.go

```diff
@@ -111,16 +113,21 @@ func printConfig() {
 
 // Get preferred outbound ip of this machine
 func GetOutboundIP() net.IP {
```

Comment:
maybe we could return the error instead of calling Fatal inside the func.
I'm happy to do it later